packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -367,7 +375,7 @@ export class TerminalFrontendContribution implements TerminalService, CommandCon
         return ref instanceof TerminalWidget ? ref : undefined;
     }
 
-    protected async openTerminal(options?: ApplicationShell.WidgetOptions): Promise<void> {
```

Comment:
I'm not sure about editing the access modifier of the method.
Instead, you can inject the `CommandRegistry` where you call the method, and use the following:

```ts
this.commandRegistry.executeCommand(TerminalCommands.NEW.id);
```

Reply:
Thanks Vince! It works nicely.